src/saving/SaveHelper.cs

```diff
@@ -239,17 +240,17 @@ public static List<string> CreateListOfSaves(SaveOrder order = SaveOrder.LastMod
     }
 
     /// <summary>
-    ///   Counts the total number of saves and how many bytes they take up
+    ///   Counts the total number of saves with the given regex expression and how many bytes they take up
     /// </summary>
-    public static (int Count, ulong DiskSpace) CountSaves(string? nameStartsWith = null)
+    public static (int Count, ulong DiskSpace) CountSaves(Regex? nameContains = null)
```

Comment:
```suggestion
    public static (int Count, ulong DiskSpace) CountSaves(Regex? nameMatches = null)
```

as you use the "IsMatch" method and don't search if the regex is matched anywhere in the string, I think this should be renamed.